t/base/protocols.t

```diff
@@ -1,20 +1,23 @@
 use strict;
 use warnings;
 use Test::More;
-plan tests => 6;
 
 use LWP::UserAgent;
+plan tests => 8;
```

Comment:
I think, in general, we no longer need to a plan.  `done_testing()` at the end of the file is fine.